src/Test/Perf/util/test_util.csx

```diff
@@ -127,14 +125,14 @@ class ProcessResult
     public bool Succeeded => !Failed;
 }
 
-/// Shells out, and if the process fails, log the error
-/// and quit the script.
+// Shells out, and if the process fails, log the error
+// and quit the script.
 void ShellOutVital(
         string file,
         string args,
         string workingDirectory = null,
         CancellationToken? cancelationToken = null) {
-    var result = ShellOut(file, args, workingDirectory, cancelationToken);
+    var result = ShellOut(file, args, workingDirectory, logStreamingOutput: false, cancelationToken: cancelationToken);
```

Comment:
I like this.  I've made this change manually, so I think it's a useful option to have.